test/data/cmumps2fhir-patient7.shex

```diff
@@ -0,0 +1,587 @@
+﻿PREFIX fhir:<http://hl7.org/fhir/> 
```

Comment:
I can't do this in the ShEx files right now.  The ShEx materialize tool appears to fail when given a file with embedded comments.   I can add a README file to the test/data directory, but that's not really a place where most people would be looking to read up on ShEx, FHIR, or CMUMPS.  How important is this?